migrate_to_cassandra.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from cqlengine.connection import LOG
+
+from scrapi import settings
+from scrapi.util.storage import store
+
+from migration_tasks import process_one_to_cassandra
+
+
+logger = logging.getLogger(__name__)
+LOG.setLevel(logging.WARN)
+
+
+def main():
+    for consumer_name in settings.MANIFESTS.keys():
+        consumer = settings.MANIFESTS[consumer_name]
+        for raw_path in store.iter_raws(consumer_name, include_normalized=True):
+            process_one_to_cassandra.delay(consumer_name, consumer, raw_path)
```

Comment:
You shouldn't be passing the consumer in here just the name, then use the import consumer function.